openapi-filter.js

```diff
@@ -47,10 +47,23 @@ let argv = require('yargs')
     .number('maxAliasCount')
     .default('maxAliasCount',100)
     .describe('maxAliasCount','maximum YAML aliases allowed')
+    .alias('configFile', 'c')
+    .describe('configFile', 'The file & path for the filter options')
     .help()
     .version()
     .argv;
 
+// apply options from config file if present
+if (argv && argv.configFile) {
+    console.log('Options Config file used: ', argv.configFile);
```

Comment:
Maybe drop this, or only output if `--verbose` is set?

Reply:
@MikeRalphson Do we have the --verbose option already? Or can I add it?

I find it handy to show the configFile that is being used, since if you have the openapi-filter running multiple times via deploy pipelines, it is handy to see which config file is being used in each step.

If you approve with the `--verbose` option, would you mind that I also show a success message when the filter went successful?

Something like

`IN File:            input.yaml`
`OUT File:        output.yaml`
`CONFIG File:  options.yaml`
`✅  OpenAPI spec was filtered successful!`

Reply:
Sorry, most of my tools have a `--verbose` option (of yargs type: `count`) so I assumed we had one here. Feel free to add.
Just a note that the logging should probably go to `console.warn` so it doesn't get mixed up with the output if you omit an output-file option.

Also, the input might not be an OpenAPI **document** (not 'spec'), but it could be AsyncAPI etc, so something less specific here please.

Reply:
I added a verbose option in this PR and took into account your remarks.

Feel free to give feedback or corrections.